src/Components/ProfilePage.js

```diff
@@ -1,41 +1,79 @@
 import React, { Component } from 'react';
 import Images from "../images.js"
-import { MDBContainer, MDBRow, MDBCol, MDBBtn, MDBIcon, MDBCardBody } from "mdbreact";
+import axios from "axios";
+import {MDBRow, MDBBtn} from "mdbreact";
 import './styles/ProfilePage.css'
+import ProfileForm from './modals/ProfileForm.js'
 
 class ProfilePage extends Component {
-    constructor(){
-        super();
+    constructor(props){
+        super(props);
         this.state = {
-            name: "Lulu L.",
-            biography: "Et harum quidem rerum facilis est et expedita distinctio. Nam libero tempore, cum soluta nobis est eligendi optio cumque nihil impedit quo minus id quod maxime placeat facere possimus, omnis voluptas assumenda est, omnis dolor repellendus. Temporibus autem quibusdam et aut officiis debitis aut rerum necessitatibus saepe eveniet ut et voluptates repudiandae sint et molestiae non recusandae. Itaque earum rerum hic tenetur a sapiente delectus, ut aut reiciendis voluptatibus maiores alias consequatur aut perferendis doloribus asperiores repellat. ",
-            program: "SYDE - 3A",
-            socialMedia: "@LuluLiu",
-            avatar: Images.avatar.chicken
-            }
+            user : this.props.userInfo,
+            token : this.props.userToken
+        }
+        this.handleLogout = this.handleLogout.bind(this);
+        this.handleProfileUpdate = this.handleProfileUpdate.bind(this);
     }
+
+    handleProfileUpdate = (content)  => {
+        this.setState({user : content});
```

Comment:
I would set user state after the request has been successfully made. Otherwise, in the failure case this will make the ui out of sync with the backend.